torch/_meta_registrations.py

```diff
@@ -4192,6 +4190,13 @@ def activate_meta():
                 _meta_lib_dont_use_me_use_register_meta_for_mkldnn.impl(op_overload, fn)
             elif "mkl::" in op_overload.name():
                 _meta_lib_dont_use_me_use_register_meta_for_mkl.impl(op_overload, fn)
+            elif "quantized_decomposed::" in op_overload.name():
+                # from torch.ao.quantization.fx._decomposed import quantized_decomposed_lib  # noqa: F401
+
+                _QUANTIZED_DECOMPOSED_LIB = torch.library.Library(
+                    "quantized_decomposed", "IMPL", "Meta"
+                )
+                _QUANTIZED_DECOMPOSED_LIB.impl(op_overload, fn)
```

Comment:
I dont follow why this change is needed? I see you removed meta imlementations and you seem to be adding those here, but what is the "fn" here?

Reply:
my understanding is that when people do register_decomposition, meta kernel is auto generated or something, but I'm planning to revert this since I want to do rewrite instead of decompositions